environment.yml

```diff
@@ -33,7 +33,8 @@ dependencies:
   - nbconvert>=5.4.1
   - nbsphinx
   - pandoc
-  # Dask and its dependencies
+
+  # Dask and its dependencies (that dont install with dask)
```

Comment:
IIUC `conda install dask` does the right thing without needing to specify everything else, but `pip install dask` does not.  could we unify all this as `dask[dataframe]`?  also i guess put it in the "downstream tests" block?

Reply:
@TomAugspurger might have ideas

Reply:
`dask[dataframe]` works for installing from PyPI, but there isn't a `dask[dataframe]` or `dask-dataframe` conda package.